.github/workflows/bats.yml

```diff
@@ -30,14 +30,18 @@ jobs:
         env:
          BATS_LIB_PATH: ${{ steps.setup-bats.outputs.lib-path }}
          TERM: xterm
-        run: bats -o _artifacts tests/
+        run: |
+          bats -o _artifacts tests/
+          bats -o _artifacts-custom-network tests/custom-network/
```

Comment:
split this into two steps, so it is clear when the job fails which one of the scenarios failed, the rest lgtm, so you can squash all commits too

Reply:
makes sense, done